test/libevmasm/Optimiser.cpp

```diff
@@ -1233,7 +1233,17 @@ BOOST_AUTO_TEST_CASE(jumpdest_removal_subassemblies)
 	main.append(t1.toSubAssemblyTag(subId));
 	main.append(u256(8));
 
-	main.optimise(true, solidity::test::CommonOptions::get().evmVersion(), false, 200);
+	Assembly::OptimiserSettings settings;
+	settings.isCreation = false;
+	settings.runInliner = false;
```

Comment:
Why don't you run it?

Reply:
Because this is a test for the jumpdest remover...
I could run it, but then I'd need to adjust the expectations of this test as well :-). I could do that...

Reply:
Ah sure, didn't see that!